tools/cust_acct_setup/aws/terraform.tfvars.sample

```diff
@@ -0,0 +1,21 @@
+account_id = "<existing AWS Account Number>"
```

Comment:
Just wondering can account ID be derived from given access automatically? maybe we can explore this for later improvements.

Reply:
there is an AWS CLI/API to do this - not sure how easy to call from TF but can certainly take up later.